helm/azure-private-endpoint-operator/Chart.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v2
+
+name: azure-private-endpoint-operator
+appVersion: "[[ .AppVersion ]]"
+version: "[[ .Version ]]"
+
+# Please describe what the app provides in a sentence or two.
+description: Operator for reconciling Azure Private Links in CAPZ workload clusters to Azure Private Endpoints in CAPZ management clusters
+
+home: https://github.com/giantswarm/azure-private-endpoint-operator
+# If you have an icon/logo, you should add it to https://github.com/giantswarm/web-assets
+# and set the final URL as a value here and uncomment.
+
+# icon: https://s.giantswarm.io/app-icons/example/1/light.svg
+annotations:
+  application.giantswarm.io/team: clippy
```

Maybe we need to change that, I guess it's official `phoenix` component?

```suggestion
  application.giantswarm.io/team: phoenix
```

---

I updated to turtles for now, while we're still finishing tests part (so we don't send noise to Phoenix). Then we can update in the end before handing off to Phoenix. WDYT?

---

Related https://github.com/giantswarm/github/pull/971